src/index.html

```diff
@@ -53,9 +53,9 @@
         <div class="col-md-4 col-sm-12">
             <div class="feature">
                 <div class="feature-icon"><img src="assets/dash/2x/multiplatform performance light op1@2x.png"/></div>
-                <div class="feature-title">Optimized<br/>for UI</div>
-                <div class="feature-desc">Develop with a programming language
-                specialized around the needs of user interface creation
+                <div class="feature-title">Approachable<br></div>
+                <div class="feature-desc">Develop with a strongly typed programming language that is consistent,
+                concise, and offers modern features like null safety and pattern matching
```

Comment:
```suggestion
                <div class="feature-desc">Develop with a consistent, concise, and strongly typed
                programming language that offers modern features like null safety and pattern matching.
```

Reply:
Could you add a period to the end of the "Productive development" description too ([line 67](https://github.com/dart-lang/site-www/pull/5413/files#diff-ce434ef5cc161194b450135a3586eb89357fc270e8aaa8e336db5a04e8d52fbeR67))? (I can't comment there since it's not part of your pr)